NEWS.md

```diff
@@ -1,5 +1,8 @@
 # ggplot2 (development version)
 
+* `geom_tile()` now correctly recognises missing data in `xmin`, `xmax`, `ymin`,
+  and `ymax` (@thomasp85, #4495)
```

Comment:
Shouldn't we credit @sigmapi as well for the fix?

Reply:
yeah, good catch